hvplot/__init__.py

```diff
@@ -55,21 +55,36 @@ def _get_doc(cls, kind, completions=False, docstring=True, generic=True, style=T
         defaults = argspec.defaults or [None]*len(args)
         for arg, dflt in zip(args, defaults):
             parameters.append((arg, dflt))
+        signature = None
     else:
         sig = inspect.signature(method)
         for name, p in list(sig.parameters.items())[1:]:
             if p.kind == 1:
                 parameters.append((name, p.default))
+
+        extra_kwargs = _hv.core.util.unique_iterator(valid_opts + kind_opts
+                                                     + converter._axis_options
+                                                     + converter._op_options)
```

Comment:
Don't have to change this now but I've adopted this indentation style (which conforms to blacks line wrapping rules) for Panel and hvPlot now:

```python        
extra_kwargs = _hv.core.util.unique_iterator(
    valid_opts + kind_opts + converter._axis_options + converter._op_options
)
```

Reply:
Addressed in bbf6f61.